syntax = "proto3";
package com.moon.demo.ss;

enum MsgID {
    PLACE_HOLDER = 0;
    MSG_ID_DEMO2_REQ = 2000;
    MSG_ID_DEMO2_RSP = 2001;
    MSG_ID_DEMO3_REQ = 2002;
    MSG_ID_DEMO3_RSP = 2003;
    MSG_ID_DEMO4_REQ = 2004;
    MSG_ID_DEMO4_RSP = 2005;
    MSG_ID_DEMO5_REQ = 2006;
    MSG_ID_DEMO5_RSP = 2007;
    MSG_ID_DEMO6_REQ = 2008;
    MSG_ID_DEMO6_RSP = 2009;
    MSG_ID_DEMO7_REQ = 2010;
    MSG_ID_DEMO7_RSP = 2011;
    MSG_ID_DEMO8_REQ = 2012;
    MSG_ID_DEMO8_RSP = 2013;
    MSG_ID_DEMO9_REQ = 2014;
    MSG_ID_DEMO9_RSP = 2015;
    MSG_ID_DEMO10_REQ = 2016;
    MSG_ID_DEMO10_RSP = 2017;
    MSG_ID_DEMO20_REQ = 2018;
    MSG_ID_DEMO20_RSP = 2019;
    MSG_ID_DEMO50_REQ = 2020;
    MSG_ID_DEMO50_RSP = 2021;
    MSG_ID_DEMO90_REQ = 2022;
    MSG_ID_DEMO90_RSP = 2023;
    MSG_ID_DEMO2_NFY = 2024;
    MSG_ID_DEMO4_NFY = 2025;
    MSG_ID_DEMO5_NFY = 2026;
}
